addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -98,17 +130,10 @@ var WysiwygMultizone = Wysiwyg.extend({
         'submit .note-editable form .btn': function (ev) {
             ev.preventDefault(); // Disable form submition in editable mode
         },
-        'hide.bs.dropdown .dropdown': function (ev) {
```

Comment:
Since you are not changing this function anymore, leave it instead of separated handler. This will ease reading the diff and will make CHM's rebase easier.